Documentation/commands.md

```diff
@@ -149,6 +149,10 @@ sha512-c4010045aec65aefa74770ef2bb648d9
 
 Docker images do not support signature verification.
 
+#### Authentication
+
+If you want to download an image from a private repository, then you will often need to pass credentials to be able to access it. rkt currently supports authentication for fetching images via https:// or docker:// protocols. To specify credentials you will have to write some configuration files. The format of configuration file and examples [are here](configuration.md). Note that there is different configuration kind for images downloaded via https:// protocol and different for ones downloaded via docker:// one.
```

Comment:
Last sentence is confusing. How about

"Note that the configuration kind for images downloaded via https:// and images downloaded via docker:// is different."